source/vibe/core/drivers/libasync.d

```diff
@@ -117,6 +119,11 @@ final class LibasyncDriver : EventDriver {
 		logTrace("Deleting event driver");
 		m_break = true;
 		getEventLoop().exit();
+		s_refCount.atomicOp!"-="(1);
+		if (s_refCount.atomicLoad() == 0) {
```

Comment:
Not important at all in this case, but you could use the return value of `atomicOp` directly to avoid another load.